fonttools-subset: add page

callumr00 opened the pull request.

Managor commented:
The base fonttools file doesn't exist. Please don't create a subcommand page until the base page is created and filled up

Managor reviewed (changes requested):
Previous comment

callumr00 commented:
I've added more examples to `fonttools-subset.md` and introduced `fonttools.md` at @Managor's request. Sorry it wasn't included initially!

`fonttools` is largely a jumping-off point for `fonttools subset`, `fonttools merge`, etc. and has no commands of it's own besides `--help`, so that's the only snippet I've included in `fonttools.md`.

callumr00 commented on pages/linux/fonttools.md:
I've now added the first example of `fonttools-subset.md` to `fonttools.md`.

dmmqz commented on pages/linux/fonttools-subset.md:
```diff
@@ -0,0 +1,20 @@
+
+- Subset a TTF font file to the Basic Latin Unicode block:
+
+`fonttools subset font.ttf --unicodes=U+0000-007F`
```
```suggestion
`fonttools subset {{path/to/font.ttf}} --unicodes=U+0000-007F`
```
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths

dmmqz commented on pages/linux/fonttools-subset.md:
```diff
@@ -0,0 +1,20 @@
+
+- Change the file type to WOFF2:
+
+`fonttools subset font.ttf --unicodes=U+0000-007F --flavor=woff2`
```
```suggestion
`fonttools subset {{path/to/font.ttf}} --unicodes=U+0000-007F --flavor=woff2`
```

dmmqz commented on pages/linux/fonttools-subset.md:
```diff
@@ -0,0 +1,20 @@
+
+- Keep only the onum (oldstyle figures) and kern (kerning) OpenType font features:
+
+`fonttools subset font.ttf --unicodes=U+0000-007F --layout-features=onum,kern`
```
```suggestion
`fonttools subset {{path/to/font.ttf}} --unicodes=U+0000-007F --layout-features=onum,kern`
```

dmmqz commented on pages/linux/fonttools-subset.md:
```diff
@@ -0,0 +1,20 @@
+
+- Set the output file's name:
+
+`fonttools subset font.ttf --unicodes=U+0000-007F --output-file=subset.ttf`
```
```suggestion
`fonttools subset {{path/to/font.ttf}} --unicodes=U+0000-007F --output-file={{path/to/subset.ttf}}`
```

dmmqz commented on pages/linux/fonttools.md:
```diff
@@ -0,0 +1,12 @@
+# fonttools
+
+> Libraries and utilities for manipulating fonts in Python.
```
```suggestion
> Manipulate fonts in Python.
```
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood. Feel free to change this, but it must be phrased in the imperative mood.

Managor replied:
This is still not fixed

callumr00 replied:
My apologies I thought I had already. I've updated it now.

dmmqz commented on pages/linux/fonttools.md:
```diff
@@ -0,0 +1,12 @@
+
+- Subset a TTF font file to the Basic Latin Unicode block:
+
+`fonttools subset font.ttf --unicodes=U+0000-007F`
```
```suggestion
`fonttools subset {{path/to/font.ttf}} --unicodes=U+0000-007F`
```

Managor commented:
>fonttools is largely a jumping-off point for fonttools subset, fonttools merge, etc. and has no commands of it's own besides --help
We treat the base pages as the overview of all the subcommands of the command. See `git.md` as an example.
